### PR TITLE
Fix AssetDescriptor

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/881

### CODE REVIEW

**Comment by vncoelho** [comment]
Master, Shargon, fast as a bolt.

File: neo/Wallets/AssetDescriptor.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public AssetDescriptor(UInt160 asset_id)
                 sb.EmitAppCall(asset_id, "name");
                 script = sb.ToArray();
             }
-            ApplicationEngine engine = ApplicationEngine.Run(script);
+            ApplicationEngine engine = ApplicationEngine.Run(script, extraGAS: 10_000_000);
```
Comment (igormcoelho) [inline neo/Wallets/AssetDescriptor.cs]: what's this value?

File: neo.UnitTests/UT_NeoToken.cs [hunk 2]
```diff
@@ -29,6 +29,15 @@ public void TestSetup()
             Store = TestBlockchain.GetStore();
         }
 
+        [TestMethod]
+        public void Check_AssetDescription()
```
Comment (erikzhang) [inline neo.UnitTests/UT_NeoToken.cs]: I think it is not related to native contracts.

File: neo/Wallets/AssetDescriptor.cs (hunk 1 above)
Comment (shargon) [reply neo/Wallets/AssetDescriptor.cs]: We can adjust to a better one

File: neo.UnitTests/UT_NeoToken.cs (hunk 2 above)
Comment (shargon) [reply neo.UnitTests/UT_NeoToken.cs]: Other ut class?

Comment (erikzhang) [reply neo.UnitTests/UT_NeoToken.cs]: `UT_AssetDescription`?

File: neo/Wallets/AssetDescriptor.cs (hunk 1 above)
Comment (devhawk) [reply neo/Wallets/AssetDescriptor.cs]: This seems arbitrary. Would setting test mode be a better approach?

Comment (erikzhang) [reply neo/Wallets/AssetDescriptor.cs]: `name()` may contain an infinite loop, and `testMode` can cause the client to fail to jump out of an infinite loop.

**Pull request merged**